app/gui/view/graph-editor/src/component/node/input/widget.rs

```diff
@@ -126,9 +138,17 @@ pub struct OverrideKey {
 
 /// Common trait for constructing and reconfiguring all widget variants. See "Widget Lifecycle"
 /// section of the module documentation for more details.
-pub trait SpanWidget {
+pub trait SpanWidget: Sized {
```

Comment:
The `Sized` bound should not be needed here; `Configuration` could accept a `?Sized` type parameter. I think it would express more clearly that `Configuration<T>` doesn't contain a `T`.